src/Microsoft.Health.Fhir.Core/Models/CodableConceptInfo.cs

```diff
@@ -12,18 +12,28 @@ namespace Microsoft.Health.Fhir.Core.Models
 {
     public class CodableConceptInfo
     {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CodableConceptInfo"/> class.
+        /// </summary>
         public CodableConceptInfo()
         {
             Coding = new List<Coding>();
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CodableConceptInfo"/> class.
+        /// </summary>
+        /// <param name="coding">The Coding collection.</param>
         public CodableConceptInfo(IEnumerable<Coding> coding)
         {
             EnsureArg.IsNotNull(coding);
 
             Coding = coding.ToList();
         }
 
+        /// <summary>
+        /// Gets the collection of Coding.
```

Comment:
nit: it think this reads better as `Gets the Coding collection` or `Gets the collection of Codings`

Reply:
Shouldn't the property name be Codings? What are your thoughts?